editor/editor_data.cpp

```diff
@@ -966,7 +966,8 @@ EditorData::EditorData() {
 	script_class_load_icon_paths();
 }
 
-///////////
+///////////////////////////////////////////////////////////////
```

Comment:
In the day and age of IDE navigation, I'm not sure if those huge separator comments are still relevant. I'd leave it as it is currently, and we should discuss whether we should keep them (or haromonize them, or remove them) in a proposal.